src/main/java/crazypants/enderio/entity/SkeletonHandler.java

```diff
@@ -0,0 +1,30 @@
+package crazypants.enderio.entity;
+
+import cpw.mods.fml.common.eventhandler.EventPriority;
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import cpw.mods.fml.common.registry.EntityRegistry;
+import crazypants.enderio.EnderIO;
+import net.minecraft.entity.monster.EntitySkeleton;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.EntityJoinWorldEvent;
+
+public class SkeletonHandler
+{
+    @SubscribeEvent(priority = EventPriority.HIGHEST)
+    public void skeletonJoinWorld(EntityJoinWorldEvent event)
+    {
+        if (event.entity.getClass() == EntitySkeleton.class && ((EntitySkeleton)event.entity).getSkeletonType()==1)
```

Comment:
No check for the config? Aren't you just spawning in an unregistered entity in that case?